### PR TITLE
Spanish: Fix Stringtable

### PR DESCRIPTION
**Pull request by ghost**
Cambié lo de cortador de "cables" por "tenazas" y "dar" por "administrar"

### CODE REVIEW

**Comment by desklet** [comment]
El merge  se me ha colado que nosé como borrarlo de ahí aún, no le hagais caso

**Comment by desklet** [comment]
arreglado

**Comment by esteldunedain** [comment]
We'll only be merging the changes in logistics. The new medical translations are too long for the action menu.

**Comment by KoffeinFlummi** [comment]
We'll also need you to add yourself to the AUTHORS file with a valid email address in your pull request (or another one).

**Comment by bux578** [comment]
https://github.com/ghost is a deleted user ^^

File: addons/logistics_wirecutter/stringtable.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
             <English>Wirecutter</English>
             <French>Pince coupante</French>
             <German>Drahtschneider</German>
-            <Spanish>Cortador de cables</Spanish>
+            <Spanish>Tenazas</Spanish>
```
Comment (ulteq) [inline addons/logistics_wirecutter/stringtable.xml]: Tenazas -> Pliers != Wirecutter, at least from what I understand.

**Pull request closed without merging**